canidropjetifier/src/main/kotlin/com.github.plnice/canidropjetifier/CanIDropJetifierTask.kt

```diff
@@ -18,12 +19,12 @@ class CanIDropJetifierTask : AllOpenTask() {
         private val OLD_MODULES_PREFIXES = listOf("android.arch", "com.android.support")
     }
 
-    var verbose: Boolean = false
-    var includeModules: Boolean = true
-    var analyzeOnlyAndroidModules = true
-    lateinit var configurationRegex: String
-    var parallelMode = false
-    var parallelModePoolSize: Int? = null
+    @Internal var verbose: Boolean = false
+    @Internal var includeModules: Boolean = true
+    @Internal var analyzeOnlyAndroidModules = true
+    @Internal lateinit var configurationRegex: String
+    @Internal var parallelMode = false
+    @Internal var parallelModePoolSize: Int? = null
```

Comment:
These should all be marked as `@Input` because they're configurable and affect the task when changed.

Reply:
Cool, I'm not expert in gradle - I put just safe annotation that will run task all the time and will make plugin compatible with the 7.x gradle

Reply:
Cool, I'm not expert in gradle - I put just safe annotation that will run task all the time and will make plugin compatible with the 7.x gradle